form.php

```diff
@@ -97,8 +97,12 @@ public function onFormProcessed(Event $event)
         switch ($action) {
             case 'captcha':
                 // Validate the captcha
+                $secret = $params['recatpcha_secret'];
+                if (empty($secret)) {
+                    $secret = $this->config->get('plugins.form.recaptcha.secret');
```

Comment:
Could not get this to save, it must be `.secret_key` not `.secret`. After you update this, good to merge.